src/fetchBaseQuery.ts

```diff
@@ -15,7 +15,7 @@ const defaultValidateStatus = (response: Response) => response.status >= 200 &&
 const isJsonContentType = (headers: Headers) => headers.get('content-type')?.trim()?.startsWith('application/json');
 
 export function fetchBaseQuery({ baseUrl }: { baseUrl?: string } = {}) {
-  return async (arg: string | FetchArgs, { signal, rejectWithValue }: QueryApi) => {
+  return async (arg: string | FetchArgs, { signal }: QueryApi) => {
```

Comment:
After trying to use this, I can't see any reason to expose `rejectWithValue` at all - we should just handle whatever error is thrown here in `buildThunks`